python/README.md

```diff
@@ -1,156 +1,3 @@
 # trezorlib
```

Comment:
could we instead symlink this to `python/docs/README.md` ? we need this file (a) for pypi page and (b) to show something nice in github view of the `python/` subfolder